packages/react/etc/react.api.md

```diff
@@ -1719,6 +1719,14 @@ export function getColorFromRGBA(rgba: IRGB): IColor;
 // @public
 export function getColorFromString(inputColor: string): IColor | undefined;
 
+// Warning: (ae-internal-missing-underscore) The name "getContextualMenuItemClassNames" should be prefixed with an underscore because the declaration is marked as @internal
```

Comment:
This warning is probably worth looking into more but I don't think it should block this.

Reply:
Asked Elizabeth.  It is a just a warning to not use it, but folks to anyway.